rconweb/api/models.py

```diff
@@ -225,7 +225,6 @@ class Meta:
                 "Can view the amount of time left in the round",
             ),
             ("can_view_server_name", "Can view the server name"),
-            ("can_view_server_stats", "Can view the get_server_stats endpoint"),
```

Comment:
I don't know enough about the permissions in Django tbh :( Do I need to do anything else here to make this change flawless? Like, will the permission automatically be removed from the groups and users that own them right now?

Reply:
I'd probably just leave the permissions in case we want to use them later, they don't really hurt anything.

But this `models.py` file needs to be updated and then we'd need to modify migration 2 or 3 (I forget) which creates the default groups

Reply:
> I'd probably just leave the permissions in case we want to use them later, they don't really hurt anything.

I don't think that this would be a very good thing. People might misinterpret the permission, especially given when it does not do anything :/

> But this models.py file needs to be updated and then we'd need to modify migration 2 or 3 (I forget) which creates the default groups

👍